### PR TITLE
Add insert_dataframe and retrieve_dataframe methods to raw

### PR DESCRIPTION
**Pull request by erlendvollset**

### CODE REVIEW

File: cognite/client/_api/raw.py [hunk 1]
```diff
@@ -492,3 +522,41 @@ def _make_columns_param(self, columns: List[str]) -> str:
...
+    def retrieve_dataframe(
+        self,
+        db_name: str,
+        table_name: str,
+        min_last_updated_time: int = None,
+        max_last_updated_time: int = None,
+        columns: List[str] = None,
+        limit: int = 25,
+    ) -> "pandas.DataFrame":
+        """Retrieve rows in a table as a pandas dataframe. <https://docs.cognite.com/api/v1/#operation/getRows>`_
```
Comment (me-ydv-5) [inline cognite/client/_api/raw.py:535]:
```suggestion
        """`Retrieve rows in a table as a pandas dataframe. <https://docs.cognite.com/api/v1/#operation/getRows>`_
```
nit

File: cognite/client/_api/raw.py [hunk 2]
```diff
@@ -346,6 +347,35 @@ def insert(
             task_unwrap_fn=lambda task: task["json"]["items"], task_list_element_unwrap_fn=lambda row: row.get("key")
         )
 
+    def insert_dataframe(self, db_name: str, table_name: str, dataframe: Any) -> None:
+        """Insert dataframe into a table <https://docs.cognite.com/api/v1/#operation/postRows>
```
Comment (me-ydv-5) [inline cognite/client/_api/raw.py:351]:
```suggestion
        """`Insert dataframe into a table <https://docs.cognite.com/api/v1/#operation/postRows>`_
```

Comment (me-ydv-5) [reply cognite/client/_api/raw.py:351]: why is this not `insert_rows` instead?

Comment (me-ydv-5) [reply cognite/client/_api/raw.py:351]: ok it's a pandas dataframe then.

File: pytest.ini [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[pytest]
+markers=
+    dsl
+    coredeps
```
Comment (me-ydv-5) [inline pytest.ini:4]: where is this used?

Comment (erlendvollset) [reply pytest.ini:4]: nowhere in this PR, but several other places. pytest emits a warning if you haven't defined the markers upfront.

**Review by me-ydv-5 — APPROVED** [review]

**Pull request merged**